org.spesmilo.Electrum.json

```diff
@@ -0,0 +1,103 @@
+{
+  "app-id": "org.spesmilo.Electrum",
+  "runtime": "org.kde.Platform",
+  "runtime-version": "5.12",
+  "sdk": "org.kde.Sdk",
+  "command": "electrum",
+  "finish-args": [
+    "--share=ipc",
+    "--socket=x11",
+    "--socket=wayland",
+    "--device=dri",
+    "--share=network",
+    "--filesystem=host",
```

Comment:
I am afraid that some Flathub maintainers may not like such high privileges.

Reply:
what do you mean do you refere to filesystem=host?